src/fiatconnect/saga.ts

```diff
@@ -731,11 +731,14 @@ export function* handleSelectFiatConnectQuote({
       crypto: parseFloat(quote.getCryptoAmount()),
       fiat: parseFloat(quote.getFiatAmount()),
     }
+    const tokenId = quote.getTokenId()
+    if (!tokenId) {
+      throw new Error(`No tokenId found in FiatConnect quote for token ${quote.getCryptoType()}`)
```

Comment:
is this safe to do if a fiat connect quote doesn't guarantee a tokenId being present in the quote?